deployments/charts/kuma/values.yaml

```diff
@@ -97,17 +97,23 @@ controlPlane:
   # TLS for various servers
   tls:
     general:
-      # -- Secret that contains tls.crt, key.crt and ca.crt for protecting Kuma in-cluster communication
+      # -- Secret that contains tls.crt, tls.key [and ca.crt when no
+      # controlPlane.tls.general.caSecretName specified] for protecting
+      # Kuma in-cluster communication
       secretName: ""
+      # -- Secret that contains ca.crt for protecting Kuma in-cluster communication
```

Comment:
this comment could be the same as for `--tls-general-ca-secret`

Reply:
addressed